src/ordered_set_lang.ml

```diff
@@ -42,14 +44,17 @@ let parse_general sexp ~f =
 
 let t =
   let open Sexp.Of_sexp in
+  context >>= fun context ->
   raw >>| fun sexp ->
   let ast =
     parse_general sexp ~f:(function
+      | Template t -> no_templates_in t.loc "here"
```

Comment:
This produces `No variables allowed in here` which looks a bit odd to me (although it might be ok?).  What about removing the second argument of `no_templates_in` and always printing `Variables are not allowed here`?

Reply:
I think the 2nd argument helps to clarify things in other cases. What about changing the function to `no_templates` and removing the "in" from the default error message.

Reply:
That seems fine.